setup.sh

```diff
@@ -0,0 +1,55 @@
+#! /bin/bash
+
+# This script sets up a Ubuntu machine to run this Node.js app locally
+
+
+# Check if node is in apt-cache
+if [ $(apt-cache show nodejs) ]
+then
+  echo "nodejs is already in the apt-cache"
+else
+  sudo apt update
+fi
+
+# Install Node.js runtime environment
+if [ -n "$(nodejs --version)" ]
+then
+  echo "Node.js is already installed"
+else
+  echo "Installing nodejs"
+  curl -fsSL https://deb.nodesource.com/setup_19.x | sudo -E bash -
+  sudo apt install -y nodejs
+fi
+
+# Install git
+if [ -n "$(git --version)" ]
+then
+  echo "git is already installed"
+else
+  echo "Installing git"
+  sudo apt-get update
+  sudo apt-get install -y git-all
+fi
+
+# Clone the repository
+if [ -d ./relativepath.tech ]
+then
+  echo "Repository is already cloned locally on system"
+else
+  echo "Cloning relativepath.tech"
+  git clone https://github.com/kensonjohnson/relativepath.tech.git
```

Comment:
Let's assume we're running this script in the checked out repo, not the other way around.

Reply:
So we don't need to do the git checkout in the script. This can be removed.

Reply:
This change has been made.